frontend/app/player/MessageDistributor/messages/urlResolve.ts

```diff
@@ -26,10 +26,9 @@ function cssUrlsIndex(css: string): Array<[number, number]> {
     const e = s + m[1].length;
     idxs.push([s, e])
   }
-  return idxs.reverse();
+  return idxs.reverse()
```

Comment:
shouldn't it be removed?

Reply:
what? Hate semicolons

Reply:
I mean .reverse(), didn't we had a bug from it?

Reply:
Yes, there was a bug when there was no reverse()